src/screens/UserPortal/Organizations/Organizations.spec.tsx

```
@@ -18,6 +19,7 @@
 import React, { act } from 'react';
 const { getItem, setItem } = useLocalStorage();
 import '../../../style/app.module.css';
+import { mocks } from 'components/AddOn/AddOnMocks';
```

ESLint

[error] 'mocks' is defined but never used.

(@typescript-eslint/no-unused-vars)